test.sh

```diff
@@ -9,6 +9,14 @@ python dynamodump/dynamodump.py -m backup -r local -s testRestoredTable --host l
   --accessKey a --secretKey a
 python tests/test.py
 
+# Test wildcard restore and backup with PAY_BY_REQUEST BillingMode
```

Comment:
This test does not actually work - it's `PAY_PER_REQUEST`.

Currently this test script is for local manual testing, thus it did not get run on CI (https://github.com/bchew/dynamodump/blob/master/.github/workflows/test.yml). I'll probably refactor this at some point so local and CI testing shares the same tests.